README.md

```diff
@@ -1,29 +1,67 @@
-# Kubernetes Template Project
+<!---
+This is an autogenerated file!
 
-The Kubernetes Template Project is a template for starting new projects in the GitHub organizations owned by Kubernetes. All Kubernetes projects, at minimum, must have the following files:
+Please do not edit this file directly, but instead make changes to the
+sigs.yaml file in the project root.
 
-- a `README.md` outlining the project goals, sponsoring sig, and community contact information
-- an `OWNERS` with the project leads listed as approvers ([docs on `OWNERS` files][owners])
-- a `CONTRIBUTING.md` outlining how to contribute to the project
-- an unmodified copy of `code-of-conduct.md` from this repo, which outlines community behavior and the consequences of breaking the code
-- a `LICENSE` which must be Apache 2.0 for code projects, or [Creative Commons 4.0] for documentation repositories, without any custom content
-- a `SECURITY_CONTACTS` with the contact points for the Product Security Team 
-  to reach out to for triaging and handling of incoming issues. They must agree to abide by the
-  [Embargo Policy](https://git.k8s.io/security/private-distributors-list.md#embargo-policy)
-  and will be removed and replaced if they violate that agreement.
+To understand how this file is generated, see https://git.k8s.io/community/generator/README.md
+--->
+# Security Special Interest Group
 
-## Community, discussion, contribution, and support
+Covers horizontal security initiatives for the Kubernetes project, including regular security audits, the vulnerability management process, cross-cutting security documentation, and security community management.
 
-Learn how to engage with the Kubernetes community on the [community page](http://kubernetes.io/community/).
```

Comment:
Link to charter doesn't exist in sig-security repo. Please update this link to https://github.com/kubernetes/community/blob/master/sig-security/charter.md